#!/usr/bin/env python
# Created on: Oct 2, 2012
#
#       Author: Pablo Inigo Blasco - Robotics and Computer Architecture (ATC) - University of Seville - 2012
#       License: GPLv3
#http://www.ros.org/wiki/dynamic_reconfigure/Tutorials/HowToWriteYourFirstCfgFile

PACKAGE = "rtcus_navigation"
import roslib;roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator import *
gen = ParameterGenerator()

gen.add("time_buffer", double_t, 0, "Samples in seconds. ", 0.2,0.01,10.0)
 
exit(gen.generate(PACKAGE, "high_noise_odometry_state_port", "HighNoiseOdometryStatePort"))

